packages/core/src/components/menubar.ts

```diff
@@ -24,7 +24,7 @@ export const MenuBar = (colors: string[]) => ({
         '@apply font-bold': {},
         '@apply gap-2 px-14 py-8': {},
         '@apply h-full min-h-full': {},
-        '@apply text-base leading-base': {},
+        '@apply leading-base': {},
```

Comment:
Why keep line-height when removing font size?

Reply:
True, I'll fix it